cep-6.md

```diff
@@ -0,0 +1,254 @@
+
+
+## Alternatives
+
+- **Show notices at the beginning of environment activation:** This was deemed too intrusive/annoying.
+- **Show notices at the beginning of command output:** Users may miss this if placed here, especially for commands
+  with lots of output
+- **Message in an HTTP header when retrieving any file from the repository:** This would be a better option for some kinds 
+  of messages, like download rate limiting or other blocks due to abuse, since it could be turned on by a rule in a CDN.
+  However, this use case is probably better addressed by having a standard way for `conda` to display errors on the 
+  console from HTTP status codes like 429 (Too Many Requests) and 403 (Forbidden). Additionally, serving custom headers
+  is challenging unless the repository owner has more control over the web server than is usually given by services 
+  like Github Pages.
+- **Add notices to the repodata.json file:** The repodata.json file is already being fetched, so adding some
+  notices would reduce the number of requests. However, the repodata.json file is way too big already, so it 
+  would require clients to download a fairly large file before even being able to show the notification.
+- **Create a generic metadata.json file containing a notices key:** This could be appealing for
+  creating a space for other kinds of channel metadata, but in order to keep things as simple as possible at the moment
+  it makes more sense to put these in their own file. Plus, it allows more flexibility for dynamic routing options
+  to this file if that becomes necessary in the future.
+
+
+## References
+
+This implementation is similar to the way that `npm` handles version update reminders:
+
+```
+npm install                                                                                                                   travishathaway@thath-work-laptop
+
+up to date, audited 365 packages in 3s
+
+34 packages are looking for funding
+  run `npm fund` for details
+
+5 vulnerabilities (1 moderate, 4 high)
+
+To address all issues (including breaking changes), run:
+  npm audit fix --force
+
+Run `npm audit` for details.
+npm notice
+npm notice New minor version of npm available! 8.5.5 -> 8.7.0
+npm notice Changelog: https://github.com/npm/cli/releases/tag/v8.7.0
+npm notice Run npm install -g npm@8.7.0 to update!
+npm notice
+```
+
+## Implementation
+
+Pull request to the full implementation:
+
+https://github.com/conda/conda/pull/11462
+
+Please add any implementation related suggestions for improvements to this pull request.
+
+## FAQ
+
+### How often will I see notices?
+
+Notices will be shown however often channel owners would like. Users will also be able to disable channel
+notices completely in order to never see them during normal operation.
+
+### Is this opt-in or opt-out?
+
+Users will automatically be opted in to these feature. They will have the ability to turn it off via
+a configuration parameter.
+
+
+## Resolution
+
+TBD
```

Comment:
Is this meant to be replaced?

Reply:
This _might_ be replaced by the voting result, but it's optional, since the [governance policy only explicitly mentions](https://github.com/conda-incubator/governance/tree/69f3ce97b3b3e7001bf6bdece7d544f2dea3a633#posting-results) documenting the results in the pull request where the vote was conducted.

I would suggest to use the section if there would have been a controversial debate prior or during the vote. In this case I think it's fine to simply remove it when marking the CEP's status as "Accepted". Just my opinion though.

Reply:
Interesting maybe we should only include this section when it is needed then?

Reply:
Yeah, that's not a bad idea, [CEP 1](https://github.com/conda-incubator/ceps/blob/main/cep-1.md#cep-content) uses "Some section that may be included if appropriate for the proposal include." so I mostly see this as part of the cleanup process before merging the proposal, in case the CEP author(s) have added it.